Running Time Entry UI card

kornelijepetak opened the pull request:
The (too) long awaited PR.

Closes #1120

willsb commented on Toggl.Giskard/Activities/MainActivity.cs:
```diff
@@ -25,6 +27,55 @@ protected override void OnCreate(Bundle bundle)
+            playButton = FindViewById(Resource.Id.MainPlayButton);
+        }
+
+        private View runningEntryCardFrame;
```
Fields go at the beginning, please

willsb commented on Toggl.Giskard/Activities/MainActivity.cs:
```diff
@@ -25,6 +27,55 @@ protected override void OnCreate(Bundle bundle)
+
+        internal void OnTimeEntryCardVisibilityChanged(bool visible)
+        {
+            
```
Extra line

willsb commented on Toggl.Giskard/Activities/MainActivity.cs:
```diff
@@ -25,6 +27,55 @@ protected override void OnCreate(Bundle bundle)
+                if (!visible)
+                    (initialRadius, finalRadius) = (finalRadius, initialRadius);
+                
+                Animator animator = ViewAnimationUtils.CreateCircularReveal(runningEntryCardFrame, cx, cy, initialRadius, finalRadius);
```
`var` :P

willsb commented on Toggl.Giskard/Activities/MainActivity.cs:
```diff
@@ -25,6 +27,55 @@ protected override void OnCreate(Bundle bundle)
+                    EventHandler onAnimationEnd = null;
+                    onAnimationEnd = (o, s) => 
+                    { 
+                        runningEntryCardFrame.Visibility = ViewStates.Invisible;
```
Where are you making this guy visible again?

kornelijepetak commented:
All tests are now passing (locally, and for bitrise, we'll see in a few minutes).
We have split the `Initialize` and `ViewAppeared` for some reason I can't recall which was causing the tests to fail.

I've reverted to the previous version of `Initialize` and `ViewAppeared` and I had no issues so I guess that was a problem with something else which was different before, but has since been changed.

It would be best that the functional test be repeated. Just in case.
